msteams-platform/concepts/build-and-test/tool-sdk-overview.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Tools and SDKs
+author: v-sdhakshina
+description: In this article, learn more about the tools and SDKs available for building your Microsoft Teams app.
+ms.localizationpriority: medium
+ms.topic: overview
+ms.author: surbhigupta
+---
+
+# Tools and SDKs
+
+Teams Platform brings you different SDKs, and APIs to build and develop your apps and there're several libraries available to support various aspects of Teams app development.
+
+The following flow diagram explains you the different SDKs, libraries and its relations:
+
+:::image type="content" source="../../assets/images/Tools-and-SDK-revamp/tools-and-sdk.png" alt-text="Screenshot shows you the list of all tools and SDKs that are available for you to build Teams apps."lightbox="../../assets/images/Tools-and-SDK-revamp/tools-and-sdk.png":::
+
+:::image type="icon" source="../../assets/icons/blue-dot.png" border="false"::: Teams developer platform SDKs require core SDKs to build apps. </br>
+:::image type="icon" source="../../assets/icons/red-dot.png" border="false"::: Teams developer platform SDKs provide a reference to core SDKs. </br>
```

Comment:
Same. Avoid the specify with the word "core". I'd suggest something like, "A library has a feature that passes your app a reference to another library."

Reply:
Updated as suggested